Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -94,6 +101,75 @@ function vtkOpenGLImageMapper(publicAPI, model) {
     shaders.Geometry = '';
   };
 
+  publicAPI.replaceShaderCoincidentOffset = (shaders, ren, actor) => {
```

Comment:
This is duplicate with OpenGL/PolyDataMapper/index.js.

Maybe you can create an OpenGL/ReplacementShaderMapper/index.js that contains a `replaceShaderCoincidentOffset = (publicAPI, model, shaders, ren, actor)` function, that is binded here ?
```
import {replaceShaderCoincidentOffset} from 'ReplacementShaderMapper/index.js';
...
publicAPI.replaceShaderCoincidentOffset = replaceShaderCoincidentOffset.bind(publicAPI, model);
```